tests/e2e/squid_ssl_bump_functionality_test.go

```diff
@@ -37,79 +119,104 @@ var _ = Describe("Squid SSL-Bump Functionality", func() {
 
 		// Get the Squid CA certificate from the ConfigMap created by trust-manager
 		By("Getting Squid CA certificate from trust-manager ConfigMap")
-		caConfigMap, err := k8sClient.CoreV1().ConfigMaps(namespace).Get(context.Background(), "proxy-ca-bundle", metav1.GetOptions{})
+		fmt.Printf("DEBUG: Retrieving proxy CA bundle from ConfigMap\n")
+		proxyCAConfigMap, err := k8sClient.CoreV1().ConfigMaps(namespace).Get(context.Background(), "proxy-ca-bundle", metav1.GetOptions{})
 		Expect(err).NotTo(HaveOccurred(), "Failed to get proxy-ca-bundle ConfigMap")
-		Expect(caConfigMap.Data).To(HaveKey("ca-bundle.crt"), "CA ConfigMap should contain 'ca-bundle.crt'")
-
-		// Create trusted client with Squid CA
-		trustedClient, err = testhelpers.NewTrustedSquidProxyClient(serviceName, namespace, []byte(caConfigMap.Data["ca-bundle.crt"]))
-		Expect(err).NotTo(HaveOccurred(), "Failed to create trusted proxy client")
-
-		// Also create regular client for comparison
-		client, err = testhelpers.NewSquidProxyClient(serviceName, namespace)
-		Expect(err).NotTo(HaveOccurred(), "Failed to create proxy client")
+		Expect(proxyCAConfigMap.Data).To(HaveKey("ca-bundle.crt"), "CA ConfigMap should contain 'ca-bundle.crt'")
+		fmt.Printf("DEBUG: Proxy CA bundle retrieved successfully\n")
+
+		// Get the test-server CA certificate from the ConfigMap created by trust-manager
+		By("Getting test-server CA certificate from trust-manager ConfigMap")
+		fmt.Printf("DEBUG: Retrieving test-server CA bundle from ConfigMap\n")
+		testServerCAConfigMap, err := k8sClient.CoreV1().ConfigMaps(namespace).Get(context.Background(), "test-server-bundle", metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred(), "Failed to get test-server-bundle ConfigMap")
+		Expect(testServerCAConfigMap.Data).To(HaveKey("ca.crt"), "Test-server CA ConfigMap should contain 'ca.crt'")
+		fmt.Printf("DEBUG: Test-server CA bundle retrieved successfully\n")
+
+		// Create trusted client with both CA bundles (same as test-client combined approach)
+		By("Creating trusted client with both CA bundles")
+		fmt.Printf("DEBUG: Creating trusted client with proxy CA and test-server CA")
+		trustedClient, err = testhelpers.NewTrustedSquidProxyClient(
+			serviceName,
+			namespace,
+			[]byte(proxyCAConfigMap.Data["ca-bundle.crt"]),
+			[]byte(testServerCAConfigMap.Data["ca.crt"]),
+		)
+		Expect(err).NotTo(HaveOccurred(), "Failed to create trusted proxy client with both CA bundles")
+		fmt.Printf("DEBUG: Trusted client created successfully\n")
 	})
 
 	Describe("SSL-Bump Certificate Inspection", func() {
-		It("should present Squid CA certificate when accessing HTTPS sites", func() {
+		It("should successfully make HTTPS request through Squid proxy with trusted client", func() {
+			// Use the helper function to create consistent HTTPS test server URL
```

Comment:
It looks like we're getting a constant rather than using a function.  No?

Reply:
Leftovers :) 
Removing it